src/commands/lambda/prompt.ts

```diff
@@ -110,6 +113,27 @@ const datadogSiteQuestion: ListQuestion = {
   type: 'list',
 }
 
+const envQuestion: InputQuestion = {
+  default: '',
```

Comment:
Can we default to undefined to avoid having to do those checks for the empty string in `setEnvServiceVersion`?

Reply:
> Looks overall good to me with one question:
> 
> Can a user instrument with interactive mode and set the forwarder instead of the extension or do we just default to setting the extension? If we do allow for the forwarder instead of the extension, we want to set tags instead of env vars

interactive mode defaults to the extension and automatically picks up the right lambda library layer depending on the runtime. If we wanted to set the forwarder in interactive mode I think we would need to add some more logic but I think that should be in a seperate pr